Sources/ZcashLightClientKit/Synchronizer/SDKSynchronizer.swift

```diff
@@ -510,8 +510,9 @@ public class SDKSynchronizer: Synchronizer {
         return try await blockProcessor.refreshUTXOs(tAddress: address, startHeight: height)
     }
 
-    public func getAccountBalance(accountIndex: Int = 0) async throws -> AccountBalance? {
-        try await initializer.rustBackend.getWalletSummary()?.accountBalances[UInt32(accountIndex)]
+    // TODO: This is hardcoded Zip32Account for index 0, must be updated
+    public func getAccountBalance(account: Zip32Account = Zip32Account(0)) async throws -> AccountBalance? {
```

Comment:
Since the type is changing incompatibly anyway, I would just take the opportunity to remove the default.
```suggestion
    public func getAccountBalance(account: Zip32Account) async throws -> AccountBalance? {
```